pkg/tests/ossm/smoke_test.go

```diff
@@ -63,6 +126,36 @@ func TestBookinfoInjection(t *testing.T) {
 	})
 }
 
+func assertProxiesReadyInLessThan10Seconds(t TestHelper, ns string) {
+	t.Log("Extracting proxy startup time and last transition time for all the pods in the namespace")
+	startedAt := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.status.containerStatuses[?(@.name=="istio-proxy")].state.running.startedAt}{"\n"}{end}`)
+	readyAt := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.status.conditions[?(@.type=="Ready")].lastTransitionTime}{"\n"}{end}`)
+
+	startedAtList := strings.Split(startedAt, "\n")
+	readyAtList := strings.Split(readyAt, "\n")
+
+	for i := 0; i < len(startedAtList) && i < len(readyAtList); i++ {
+		startedAtLine := startedAtList[i]
+		readyAtLine := readyAtList[i]
+		if startedAtLine == "" || readyAtLine == "" {
```

Comment:
If `startedAt` or `readyAt` are empty, doesn't it mean that something is wrong with pods and we should retry fetching these values?

Reply:
making the request of all the pods we will get rid of this, because the only pod without proxy is the sleep pod. So we can skip that pod verification